packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.module.scss

```diff
@@ -0,0 +1,119 @@
+@use '@/scss/underscore' as _;
+@use '@/scss/dimensions' as dim;
+
+.currentTenantCard {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: _.unit(1);
+  margin-left: - _.unit(1);
+  max-width: dim.$tenant-dropdown-max-width;
+  border-radius: _.unit(2);
+  transition: background-color 0.2s ease-in-out;
+  position: relative;
+  border: none;
+  background-color: transparent;
+
+  &:hover {
+    cursor: pointer;
+    background: #5d34f214;
+  }
+
+  &:not(:disabled) {
+    cursor: pointer;
+  }
+
+  .name {
+    font: var(--font-title-2);
+    margin-right: _.unit(1.5);
+    @include _.text-ellipsis;
+  }
+
+  .tag {
+    font: var(--font-body-3);
+    margin-right: _.unit(2);
+  }
+
+  .arrowIcon {
+    width: 16px;
+    height: 16px;
+    color: var(--color-text-secondary);
+  }
+}
+
+.dropdown {
+  max-width: dim.$tenant-dropdown-max-width;
+  min-width: dim.$tenant-dropdown-min-width;
+}
+
+.dropdownItem {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: _.unit(2.5) _.unit(3) _.unit(2.5) _.unit(4);
+  margin: _.unit(1);
+  border-radius: _.unit(1.5);
+  transition: background-color 0.2s ease-in-out;
+
+  &:hover {
+    background: var(--color-hover);
+  }
+
+  &:not(:disabled) {
+    cursor: pointer;
+  }
+
+  .dropdownName {
+    font: var(--font-body-2);
+    margin-right: _.unit(2);
+    @include _.text-ellipsis;
+  }
+
+  .dropdownTag {
+    font: var(--font-body-3);
+    margin-right: _.unit(4);
+  }
+
+  .checkIcon {
+    width: 20px;
+    height: 20px;
+    flex-shrink: 0;
+    color: transparent;
+    margin-left: auto;
+
+    &.visible {
+      color: #5d34f2;
+    }
+  }
+}
+
+.createTenantButton {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: _.unit(2.5) _.unit(3) _.unit(2.5) _.unit(4);
+  margin: _.unit(1);
+  border-radius: _.unit(1.5);
+  transition: background-color 0.2s ease-in-out;
+
+  &:hover {
+    background: var(--color-hover);
+  }
+
+  &:not(:disabled) {
+    cursor: pointer;
+  }
+
+  .createTenantButtonText {
+    font: var(--font-body-2);
+    @include _.text-ellipsis;
+  }
+
+  .createTenantButtonIcon {
+    width: _.unit(5);
+    height: _.unit(5);
+    color: var(--color-placeholder);
+    position: absolute;
+    right: _.unit(4);
+  }
```

Comment:
1. The class name can be shorter, maybe just `.icon`
2. `width` and `height` can just use pixel values, since we only use `_.unit()` to control margins and paddings
3. Why do you use absolute position for the icon?

Reply:
The icon should stick to the right side of the button.